uber/tests/email_tests/test_automated_email_categories.py

```diff
@@ -1,4 +1,4 @@
-from uber.tests.email.email_fixtures import *
+from uber.tests.email_tests.email_fixtures import *
```

Comment:
The `uber.tests.email` package was shadowing the python `email` package, and causing the requests library to fail (which is used by the test data import script).